File: front_end/ios/VideoShaping.m
//
//  VideoShaping.m
//  mashtub
//
//  Created by Dev on 2019-06-21.
//  Copyright © 2019 Facebook. All rights reserved.
//

#import "VideoShaping.h"

@implementation VideoShaping
RCT_EXPORT_MODULE()

RCT_EXPORT_METHOD(greetUser:(NSString *)name isAdmin:(BOOL *)isAdmin calback:
                  (RCTResponseSenderBlock) callback) {
  NSLog(@"User Name: %@ , Administrator: %@", name, isAdmin ? @"Yes": @"No");
  
  NSString *greeting = [NSString stringWithFormat:@"Welcome %@, you %@ an administrator", name, isAdmin ? @"are": @"are not"];
  
  callback(@[greeting]);
}
@end

File: front_end/ios/VideoShaping.swift
//
//  VideoShaping.swift
//  mashtub
//
//  Created by MOHAMED ASHIF on 21/6/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

import Foundation

@objc(VideoShaping)
class VideoShaping: NSObject {
  @objc
  func constantsToExport() -> [AnyHashable : Any]! {
    return ["initialCount": 0]
  }
}
